raiden/transfer/log.py

```diff
@@ -178,12 +201,21 @@ def log(self, state_change):
         serialized_data = self.serializer.serialize(state_change)
         self.storage.write_transaction(serialized_data)
 
+    def log_events(self, events):
+        """ Log the events into the write ahead Log. They are always assumed to be
+        events coming out of the last logged state change"""
+        assert isinstance(events, list)
+
+        id = self.last_state_change_id()
+        for event in events:
+            self.storage.write_state_event(id, self.serializer.serialize(event))
```

Comment:
The batch interface is still a TODO and not yet done. But the execute_many here is not a bad idea. Let me see what can be done.